OpenRA.Mods.Common/Widgets/Logic/Settings/InputSettingsLogic.cs

```diff
@@ -68,7 +68,9 @@ Func<bool> InitPanel(Widget panel)
 
 			var mouseScrollDropdown = panel.Get<DropDownButtonWidget>("MOUSE_SCROLL_TYPE_DROPDOWN");
 			mouseScrollDropdown.OnMouseDown = _ => ShowMouseScrollDropdown(mouseScrollDropdown, gs);
-			mouseScrollDropdown.GetText = gs.MouseScroll.ToString;
+#pragma warning disable IDE0200 // Remove unnecessary lambda expression
```

Comment:
Maybe note in the comment that `gs.MouseScroll` may change, so must be resolved at render-time?

Reply:
Added some comments.